pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -418,6 +420,33 @@ func buildInboundLocalityLbEndpoints(bind string, port int) []endpoint.LocalityL
 	}
 }
 
+func generateInboundPassthroughClusters(env *model.Environment) []*apiv2.Cluster {
+	clusters := make([]*apiv2.Cluster, 0, 2)
+	inboundPassthroughClusterIpv4 := buildDefaultPassthroughCluster(env)
+	inboundPassthroughClusterIpv4.Name = util.InboundPassthroughClusterIpv4
+	inboundPassthroughClusterIpv4.UpstreamBindConfig = &core.BindConfig{
+		SourceAddress: core.SocketAddress{
+			Address: util.InboundPassthroughBindIpv4,
+			PortSpecifier: &core.SocketAddress_PortValue{
+				PortValue: uint32(0),
+			},
+		},
+	}
+
+	inboundPassthroughClusterIpv6 := buildDefaultPassthroughCluster(env)
```

Comment:
will this get a mixer filter applied?

Reply:
I think so. Mixer filter is added in buildVirtualInboundFilter